interface/src/app/analyze/analysis/analysis.js

```diff
@@ -38,12 +48,7 @@ function AnalysisCtrl($scope, $log, $q, $state, SampleBin) {
       {'typename': 'medium'},
       {'typename': 'temperature'},
       {'typename': 'treatment'}
-    ],
-    updateMlModel: function(value) {
-      if (value !== null && value.id !== null) {
-        SampleBin.getActivityForSampleList(value.id);
-      }
-    }
+    ]
```

Comment:
This is what I'm investigating. Never seen it before and it is not happening on my local instance. I'm not seeing any changes in the code directly related to that `detail.showing` error in this PR, so I'm looking at some more comprehensive diffs to see what has changed.